src/main/scala/mesosphere/marathon/upgrade/DeploymentActor.scala

```diff
@@ -123,51 +123,35 @@ private class DeploymentActor(
   def scaleRunnable(runnableSpec: RunSpec, scaleTo: Int,
     toKill: Option[Iterable[Instance]],
     status: DeploymentStatus): Future[Unit] = {
-    val runningTasks = taskTracker.specInstancesLaunchedSync(runnableSpec.id)
-    def killToMeetConstraints(notSentencedAndRunning: Iterable[Instance], toKillCount: Int) = runnableSpec match {
-      case app: AppDefinition =>
-        Constraints.selectTasksToKill(app, notSentencedAndRunning, toKillCount)
-      case pod: PodDefinition =>
-        // TODO(PODS): Tasks to kill for pods
-        Iterable.empty[Instance]
+    val runningInstances = instanceTracker.specInstancesLaunchedSync(runnableSpec.id)
+    def killToMeetConstraints(notSentencedAndRunning: Iterable[Instance], toKillCount: Int) = {
```

Comment:
This affects a lot of places, since this value is given via the GroupManager API.
I will make this a separate task.